contracts/HATVault.sol

```diff
@@ -492,13 +492,6 @@ contract HATVault is IHATVault, ERC4626Upgradeable, OwnableUpgradeable, Reentran
     /** @notice See {IHATVault-withdrawRequest}. */
     function withdrawRequest() external nonReentrant {
         IHATVaultsRegistry.GeneralParameters memory generalParameters = registry.getGeneralParameters();
-        // require withdraw to be at least withdrawRequestEnablePeriod+withdrawRequestPendingPeriod
-        // since last withdrawRequest (meaning the last withdraw request had expired)
-        // solhint-disable-next-line not-rely-on-time
-        if (block.timestamp <
```

Comment:
so are we sure we want to remove this? could maybe be nicer for the user to leave it

Reply:
Idk this can be done in the front end so maybe better remove it to save gas

Reply:
https://github.com/hats-finance/hats-contracts/issues/366

Reply:
in https://github.com/hats-finance/hats-contracts/issues/366 I argue that we need to keep this check